examples/src/main/java/tachyon/examples/MasterConcurrencyPerformance.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.examples;
+
+import java.util.List;
+import java.util.Map;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+
+import tachyon.TachyonURI;
+import tachyon.client.file.TachyonFileSystem;
+import tachyon.util.io.PathUtils;
+
+/**
+ * Contains performance test cases where a bunch of Tachyon clients concurrently
+ * operates on Tachyon master.
+ */
+public final class MasterConcurrencyPerformance {
+  private static enum OperationType {
+    CREATE_EMPTY_FILE,
+  }
+
+  private static interface OperationCallable {
+    /**
+     * What this operation will do on the path.
+     *
+     * @param path the path this operation operates on
+     * @throws Exception when any unexpected exception happens
+     */
+    void call(TachyonURI path) throws Exception;
+  }
+
+  /** Registry where type of operation is mapped to concrete operation */
+  private static Map<OperationType, OperationCallable> sOperationRegistry = Maps.newHashMap();
+
+  static {
+    final TachyonFileSystem tfs = TachyonFileSystem.get();
+
+    /**
+     * New operation types should first be added to {@link OperationType}, and then register the
+     * {@link OperationCallable} for this type below in this static block, just use {@link #tfs}
+     * when {@link TachyonFileSystem} is needed.
+     */
+    sOperationRegistry.put(OperationType.CREATE_EMPTY_FILE, new OperationCallable() {
+      @Override
+      public void call(TachyonURI path) throws Exception {
+        tfs.create(path);
```

Comment:
Oh seems like that utility method was lost. I don't think this create will complete the file, but I guess it doesn't matter too much for the perf benchmark.